templates/master/01-master-container-runtime/_base/files/crio.yaml

```diff
@@ -159,9 +159,9 @@ contents:
     # Path to OCI hooks directories for automatically executed hooks.
     # Note: the default is just /usr/share/containers/oci/hooks.d, but /usr is immutable in RHCOS
     # so we add /etc/containers/oci/hooks.d as well
+    # /usr/share/containers/oci/hooks.d is not available in recent CRI-O
     hooks_dir = [
         "/etc/containers/oci/hooks.d",
```

Comment:
Since we already have `/etc/containers/oci/hooks.d` as a default in cri-o (as of https://github.com/cri-o/cri-o/commit/16beb92233cf04ef23757d709800586466454f3e), I'd just comment out all of hooks_dir so we can allow cri-o to set the default as it sees fit

Reply:
I don't think its a good idea. I'd rather have MCD be the single source of CRI-O configs rather then relying on CRI-O defaults. Other components might use this config to set their own hooks. Also CRI-O might be tested with `oci-systemd-hook` installed (which is not the case for FCOS and currently happening on RHCOS), so the test results from CRI-O suite might be inconvincing

Reply:
per conversations with @cgwalters we decided to keep the templated configs as slim as possible. If other components use this value, they can import the value from `cri-o/cri-o/config`, which we keep public for that very use case. Removing /usr/share/containers/oci/hooks.d will remove the oci-systemd-hook anyway (as it's only shipped there, and also being removed on rhcos), so after this PR the tests will be the same, regardless of whether this field is commented out or not.

my comment isn't necessarily blocking, but we've set the idiom in the other fields to use cri-o defaults when possible, and in this situation it is possible, so I'm still pushing to follow it

Reply:
Okay, in that case it should be done in a different PR (and moved to shared masters/workers templates)